src/components.js

```diff
@@ -826,14 +827,14 @@ const components = [
     name: 'Fixed',
     type: 'div',
     props: {},
-    style: props => ({
+    style: props => Object.assign({
       position: 'fixed',
       top: props.top ? 0 : null,
       right: props.right ? 0 : null,
       bottom: props.bottom ? 0 : null,
       left: props.left ? 0 : null,
       zIndex: props.z
-    }),
+    }, center(props)),
```

Comment:
Might be cleaner to return a full set of `top`, `right`, `bottom`, and `left` values for all props with a single function rather than assigning